plasma/client/client.py

```diff
@@ -0,0 +1,61 @@
+import json
+import rlp
+from web3.contract import ConciseContract
+from web3 import HTTPProvider
+from plasma.config import plasma_config
+from plasma.root_chain.deployer import Deployer
+from plasma.child_chain.transaction import Transaction, UnsignedTransaction
+from .child_chain_service import ChildChainService
+
+
+class Client(object):
+
+    def __init__(self, root_chain_provider=HTTPProvider('http://localhost:8545'), child_chain_url="http://localhost:8546/jsonrpc"):
+        deployer = Deployer(root_chain_provider)
+        abi = json.load(open("contract_data/RootChain.json"))
+        self.w3 = deployer.w3
```

Comment:
Client.w3 is never used after this, could just replace with `self.root_chain = deployer.w3.eth...`